src/auth/__tests__/AuthProvider.unit.test.tsx

```diff
@@ -4,11 +4,17 @@ import { AuthProvider } from "../AuthProvider";
 import { render } from "@testing-library/react";
 import { HookResult, renderHook, act } from "@testing-library/react-hooks";
 import mockAxios from "jest-mock-axios";
-import { makeTestError, testAdmin, testUser } from "../../shared/test-utils";
+import {
+  makeTestError,
+  testAdmin,
+  testUser,
+  MOCK_ERROR,
+} from "../../shared/test-utils";
 import ERRORS from "../../shared/errors";
 
 // for spies
 import * as _a from "../Auth";
+import { APIResponseFailure, APIError } from "../../shared/types";
```

Comment:
Should be moved up --- this import is not for spies

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/33/commits/ecdce765bc5cda7b620792b79b5cd69050532277